app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -175,6 +175,14 @@ def firmware_types
       }
     end
 
+    def disk_mode_types
+      {
+        "persistent" => N_("Persistent"),
```

Comment:
This isn't translated correctly.
`N_()` just marks a string for translation but doesn't actually translate it.
Please use either `_()` or translate it in the view.
```ruby
<%= select_f f, :mode, compute_resource.disk_mode_types, :first, ->(d) { _(d.last) }, { }, :class => "span5", :label => _("Disk mode"), :label_size => "col-md-2", :disabled => !new_host %>
```